### ISSUE TITLE
Racket v7.9 and below download links gone

### ISSUE THREAD

**Issue by shhyou** [racket/racket-lang-org#336]
The download link of v7.9 is hidden. Although "More Installers and Checksums" still work, the download button and "or mirror" is missing. This appears to be v7.9 specific.

<https://download.racket-lang.org/racket-v7.9.html>
![Screenshot of Racket v7.9 download page where the direct download button is hidden](https://github.com/user-attachments/assets/2cb5c275-a682-4003-99e6-0a4e835ac76f)

The installers still exist on the server, however:
![Screenshot of Racket v7.9 mirror link](https://github.com/user-attachments/assets/810fbdd7-fc4f-423c-8fb7-be3921a5b126)

On a side, I'm not sure why, but v7.9 and below link to minimal Racket instead of the "all installers" page.
<https://download.racket-lang.org/all-versions.html>
![Screenshot of Racket all-versions download page where v7.9 and below suggest minimal Racket instead](https://github.com/user-attachments/assets/c8c811f3-26c0-411c-b783-74fba3f618eb)

**Comment by sorawee** [comment]
The first mentioned issue is a bug, and that's fixed in the PR linked above.

IIRC, previously, "Racket" and "Minimal Racket" are the two links. Then, at some point, we decided to no longer offer the "Minimal Racket" link (even as a drop-down item in the main download page). So we use "All Installers" in place instead. I do  think it might be a good idea to just use "All Installers" consistently for all versions. That being said, there's nothing wrong per se with the current output. It's by design.

**Comment by sorawee** [comment]
Fix deployed!

CC @jbclements in case he agrees that we should switch to use "All Installers" consistently in all versions.

**Comment by jbclements** [comment]
Yes, this change seems like a good one to me. It looks like it won’t work for versions prior to 5.92, unless the URL lives in a place I’m not finding it.

Cc:ing racket release management… any opinions on this? @sorawee, I think we should wait for three days, and then go ahead and make the change if no one objects?

John
